snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -513,53 +499,56 @@ func (b *Bootstrapper) Ancestors(ctx context.Context, nodeID ids.NodeID, request
 			zap.Stringer("expectedBlkID", wantedBlkID),
 			zap.Stringer("blkID", actualID),
 		)
+		b.PeerTracker.RegisterFailure(nodeID)
 		return b.fetch(ctx, wantedBlkID)
 	}
 
-	blockSet := make(map[ids.ID]snowman.Block, len(blocks))
+	var (
+		numBytes  = len(requestedBlock.Bytes())
+		ancestors = make(map[ids.ID]snowman.Block, len(blocks))
+	)
 	for _, block := range blocks[1:] {
-		blockSet[block.ID()] = block
+		numBytes += len(block.Bytes())
+		ancestors[block.ID()] = block
 	}
-	if err := b.process(ctx, requestedBlock, blockSet); err != nil {
+
+	// TODO: Calculate bandwidth based on the blocks that were persisted to
+	// disk.
+	var (
+		requestLatency = time.Since(requestTime).Seconds() + epsilon
```

Comment:
If we avoid this cast to `Seconds` and track all of our bandwidth in bytes/nsec can't we avoid the usage of epsilon to guard against division by zero? I think this is also more consistent with our other timing metrics which are all in nanoseconds.

Reply:
> can't we avoid the usage of epsilon to guard against division by zero?

I don't think so. That would just make it an integer div by 0 rather than a floating point div by 0 (which would panic rather than returning NaN or inf.

fwiw I don't think this should really matter... But I do know we've seen unit test flakes on windows when we expect `time.Now() - time.Now() != 0`.

As far as using nanoseconds vs seconds... I'm not sure that we have any metrics that are bytes per nanosecond... Do we? (also this isn't a metric - these values are actually used).